instructor/common.py

```diff
@@ -261,10 +261,6 @@ def assignments_variables(request):
     if request.method == "POST":
         form = AssignmentForm(request.POST, instance=assignment)
         if form.is_valid():
-            # If form has changed want to delete existing Treatments
```

Comment:
If I uncheck something like 'Collection Time' for example, and I previously had two collection times defined, the number of treatment combinations in `edit_strain_treatments` should shrink by half since there aren't two collection times anymore. Is there a way to do this easily?